haystack/retriever/squad_to_dpr.py

```diff
@@ -140,7 +140,7 @@ def add_is_impossible(squad_data: dict, json_file_path: Path):
 
     squad_data["data"] = squad_articles
     with open(new_path, "w") as filo:
```

Comment:
Why dont we use utf-8 encoding here?

Reply:
Maybe we can change all file open encoding to utf-8?